constraint/pkg/apis/templates/v1/scheme.go

```diff
@@ -0,0 +1,42 @@
+package v1
+
+import (
+	ctschema "github.com/open-policy-agent/frameworks/constraint/pkg/schema"
+	"k8s.io/apiextensions-apiserver/pkg/apis/apiextensions"
+	apiextensionsv1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
+	"k8s.io/apiextensions-apiserver/pkg/apiserver/schema"
+	"k8s.io/apimachinery/pkg/runtime"
+)
+
+const version = "v1"
+
+var (
+	structuralSchema *schema.Structural
+	sch              *runtime.Scheme
+)
+
+func init() {
+	// Prevent problems with ordering of init() function calls.  These
+	// functions are called according to the lexicographic order of their
+	// containing files.  As Register() is called on the localSchemeBuilder by
+	// zz_generated.conversion.go, the conversion functions haven't been
+	// registered with the localSchemeBuilder by the time this init() function
+	// runs.  We sidestep this problem by adding RegisterConversions here.
+	schemeBuilder := runtime.NewSchemeBuilder(localSchemeBuilder...)
+	schemeBuilder.Register(RegisterConversions)
```

Comment:
We should create a scheme builder that explicitly registers all necessary registrations. Otherwise this function will still be vulnerable to failures due to reordering that changes the contents of `localSchemeBuilder`

Reply:
Define the schemeBuilder outside of a function as a global var to avoid ordering concerns:

```
var schemeBuilder = runtime.NewSchemeBuilder(fn1, fn2, fn3, ...)
```

Reply:
It's okay to have our own package-internal scheme,
```
var toVersionlessSchemeBuilder = runtime.NewSchemeBuilder(...)
```